packages/bpk-foundations-common/base/colors/aliases.json

```diff
@@ -1,73 +1,80 @@
-    "BLACK_TINT_01": "#1D1B20",
-    "BLACK_TINT_02": "#2C2C2E",
-    "BLACK_TINT_03": "#3A3A3C",
-    "BLACK_TINT_04": "#48484A",
-    "BLACK_TINT_05": "#636366",
-    "BLACK_TINT_06": "#8E8E93",
+    "SKY_BLUE": "#0062E3",
+    "SKY_BLUE_85": "#024DAF",
+    "MONTEVERDE": "#0C838A",
+    "GLENCOE": "#62F1C6",
+    "SAGANO": "#D4FFF2",
+    "KOLKATA": "#F55D42",
+    "ERFOUD": "#FEEB87",
+    "BAGAN": "#FFF7CF",
+    "BUNOL": "#F55D42",
+    "PETRA": "#FEEB87",
+    "NARA": "#FFF7CF",
+    "ABISKO": "#05203C",
+    "VALENSOLE": "#E0E3E5",
+    "TOCHIGI": "#EFF1F2",
+    "PANJIN": "#E70866",
+    "HILLIER": "#FF649C",
+    "HARBOUR": "#FFE9F9",
+    "SKY_BLUE_SHADE_03": "#05203C",
+    "SKY_BLUE_SHADE_02": "#05203C",
+    "SKY_BLUE_SHADE_01": "#0062E3",
+    "SKY_BLUE_TINT_01": "#84E9FF",
+    "SKY_BLUE_TINT_02": "#E0E3E5",
+    "SKY_BLUE_TINT_03": "#E0E3E5",
+    "SKY_GRAY_TINT_07": "#EFF1F2",
+    "SKY_GRAY_TINT_06": "#E0E3E5",
+    "SKY_GRAY_TINT_05": "#C2C9CD",
+    "SKY_GRAY_TINT_04": "#C2C9CD",
+    "SKY_GRAY_TINT_03": "#545860",
+    "SKY_GRAY_TINT_02": "#545860",
+    "SKY_GRAY_TINT_01": "#545860",
+    "SKY_GRAY": "#161616",
+    "BLACK": "#010913",
+    "BLACK_TINT_01": "#131D2B",
+    "BLACK_TINT_02": "#131D2B",
+    "BLACK_TINT_03": "#243346",
+    "BLACK_TINT_04": "#44505F",
+    "BLACK_TINT_05": "#44505F",
+    "BLACK_TINT_06": "#BDC4CB",
     "PRIMARY_GRADIENT_LIGHT": "#02DDD8",
-    "DARK_SKY": "#02122C",
+    "DARK_SKY": "#05203C",
+    "DARK_SKY_85": "#154679",
     "ECO_GREEN": "#0FA1A9",
-    "GREEN_SPOT": "#00A698",
-    "GREEN_MUTED": "#D0EEEC",
-    "AMBER_SPOT": "#FF9400",
-    "YELLOW_MUTED": "#FFEBD0",
-    "BERRY": "#D1435B",
-    "PINK_MUTED": "#F6DDE1",
-    "GREY_10": "#F1F2F8",
-    "GREY_20": "#DDDDE5",
-    "GREY_30": "#CDCDD7",
-    "GREY_40": "#68697F",
-    "CHARCOAL": "#111236",
+    "GREEN_SPOT": "#0C838A",
+    "GREEN_MUTED": "#D4FFF2",
+    "AMBER_SPOT": "#F55D42",
+    "YELLOW_MUTED": "#FFF7CF",
+    "BERRY": "#E70866",
+    "BERRY_75": "#C80456",
+    "PINK_MUTED": "#FFE9F9",
+    "GREY_10": "#EFF1F2",
+    "GREY_20": "#E0E3E5",
+    "GREY_30": "#C2C9CD",
+    "GREY_40": "#545860",
+    "CHARCOAL": "#161616",
     "NIGHT_SKY": "#054184",
-    "NIGHT_BLUE": "#6D9FEB",
-    "NIGHT_GREEN_SPOT": "#00A698",
+    "NIGHT_BLUE": "#84E9FF",
+    "NIGHT_BLUE_20": "#D1F7FF",
+    "NIGHT_BLUE_100": "#002B5B",
+    "NIGHT_GREEN_SPOT": "#62F1C6",
     "NIGHT_GREEN_FILL": "#B1FFE7",
-    "NIGHT_YELLOW_SPOT": "#FFB54D",
+    "NIGHT_YELLOW_SPOT": "#FEEB87",
     "NIGHT_YELLOW_FILL": "#FBF1BB",
-    "NIGHT_BERRY": "#D1435B",
+    "NIGHT_BERRY": "#FF649C",
     "NIGHT_PINK": "#FFCADD",
-    "NIGHT_GREY_10": "#000000",
-    "NIGHT_GREY_20": "#1D1B20",
-    "NIGHT_GREY_25": "#2C2C2E",
-    "NIGHT_GREY_30": "#48484A",
-    "NIGHT_GREY_40": "#8E8E93",
-    "WHITE_ALPHA_20": "#8E8E93",
-    "WHITE_ALPHA_80": "#8E8E93",
-    "BLACK_ALPHA_20": "#B2B2BF",
-    "BLACK_ALPHA_70": "#02122c"
+    "NIGHT_GREY_00": "#0B121D",
```

Comment:
Can we check with Adam on this one as I know he removed some aliases recently and this was one as we didn't need some of the additional ones